sedai/README.md

```diff
@@ -0,0 +1,47 @@
+# [Sedai][1] — Find out why production is the new playground with a free 14 day trial
+## Continuous Autonomous Availability Management (CAAM)
+
+*Intelligent enough to detect, independent enough to act — that's the power of autonomous*
+
+## Overview
```

Comment:
Under the Overview tab, I'd make it clear that Sedai is offering an integration with Datadog, and that if you're already a customer of Sedai you can set up the integration quickly by following the instructions in the Configuration tab.

Reply:
done